src/averageAges.js

```diff
@@ -15,17 +15,21 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(x => x.sex === 'm');
+  const averagedAgeMen = men.reduce((a, b) =>
+    a + (b.died - b.born) / men.length, 0);
```

Comment:
```suggestion
  const averagedAgeMen = men.reduce(
    (a, b) => a + (b.died - b.born) / men.length,
    0,
  );
```
или
```suggestion
  const averagedAgeMen = men
    .reduce((a, b) => a + (b.died - b.born) / men.length, 0);
```
a, b не подходит


и все в таком духе